db/migrate/20230412013301_create_account_requests.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAccountRequests < ActiveRecord::Migration[7.0]
+  def change
+    create_table :account_requests do |t|
+      t.string :name
+      t.string :fullname
+      t.string :email
+      t.string :status
+      t.text :self_introduction
```

Comment:
make it an `introduction` instead of `self_introduction`
use snake case for `fullname`